packages/config/scripts/checkVerifiedContracts/tasks.ts

```diff
@@ -7,18 +7,14 @@ import { VerificationMap } from './output'
 export async function verifyContracts(
   addresses: EthereumAddress[],
   previouslyVerified: Set<EthereumAddress>,
+  mamnuallyVerified: Set<EthereumAddress>,
```

Comment:
typo. I recommend installing spell check plugin if you use vscode :)

Reply:
thx! fixed.